inputfiles/removedTypes.jsonc

```diff
@@ -705,6 +705,20 @@
                     }
                 }
             },
+            "RTCPeerConnectionIceEventInit": {
+                "members": {
+                    "member": {
+                        "url": null // Safari +12 only
+                    }
+                }
+            },
+            "FetchEventInit": {
+                "members": {
+                    "member": {
+                        "replacesClientId": null // Removed from all browsers
```

Comment:
Not sure it's removed or simply not implemented. Unless there's clues let's say not implemented. And please write the confirmed date as things change.